### DISCUSSION TITLE
The Boolean Field property of the IIsDeletedRow interface is inconsistent with the code field type generated by Sergen (Sergen generated type is Int16Field)

### DISCUSSION THREAD

**Discussion by 15937823** [serenity-is/Serenity#6790] [General]
The Boolean Field property of the IIsDeletedRow interface is inconsistent with the code field type generated by Sergen (Sergen generated type is Int16Field)

**Comment by VictorTomaili** [comment]
Checked and there is no issue related to this. Update sergen and ensure your interface.
Ensure about the latest version of the serene, serenity and sergen. 
Please provide the version numbers and code sample which cause this problem.
